src/python/foglamp/core/server.py

```diff
@@ -5,23 +5,44 @@
 # FOGLAMP_END
 
 """Core server module"""
-
+import os
 import signal
 import asyncio
+import setproctitle
+import sys
+import time
+import requests
+import subprocess
+import socket
 from aiohttp import web
+from multiprocessing import Process
 
 from foglamp import logger
 from foglamp.core import routes
+from foglamp.core import routes_core
 from foglamp.core import middleware
 from foglamp.core.scheduler import Scheduler
+from foglamp.core.service_registry import service_registry, instance
 
-__author__ = "Praveen Garg, Terris Linenbach"
+__author__ = "Praveen Garg, Terris Linenbach, Amarendra K Sinha"
 __copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
 _LOGGER = logger.setup(__name__)  # logging.Logger
 
+_FOGLAMP_ROOT = os.getenv('FOGLAMP_ROOT', '/home/asinha/Development/FogLAMP')
+_STORAGE_PATH = '.' #os.path.expanduser(_FOGLAMP_ROOT+'/services/storage')
+
+_MANAGEMENT_PID_PATH = os.getenv('MANAGEMENT_PID_PATH', os.path.expanduser('~/var/run/management.pid'))
```

Comment:
@amarendra-dianomic I see that PID files are being used for process management. @MarkRiddoch please clarify how much/little process management should be done here.

Reply:
We can get rid of all the pid files and the like from within the code as snap has its own way of dealing with this. In non-san environments we will wrap it up inside init.d scripts